test/python/result/generate_data.py

```diff
@@ -0,0 +1,294 @@
+# This code is part of Qiskit.
```

Comment:
Can we just delete this module and integrate the test data into the test class directly. This will never be executed as far as I can tell and if we need dynamic data it should be integrated directly in the test class as a setup fixture.

Reply:
You are right. The testing framework will be changed in a following PR.

Reply:
Then lets delete this and do that we can do the larger test refactor in the follow up PR. But we don't need to carry around code that isn't used.